src/referentielijsten/conf/base.py

```diff
@@ -1,18 +1,20 @@
 import os
-from pathlib import Path
```

Comment:
`Pathlib` is considered more modern than `os.path`, so I think replacing it back was unnecessary